NuGet.config

```diff
@@ -2,7 +2,8 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-master/api/v3/index.json" />
+    <add key="AspNetCoreRelease" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />    
```

Comment:
I don't like going against the aspnetcore-ci-release feed - it feels a little bit backwards and we are risking picking up incorrect dependencies accidentally if they showed up there. Given that we need this feed only for CoreFxLabs packages - wouldn't it be better to have a separate feed for these packages instead of mirroring them to aspnet-ci-release and aspnet-ci-dev?
I need to talk to someone about source packages - I can see some of them were updated 2 days ago. I need to understand what is the process of updating these packages and how we make sure that we don't end up with a version that no longer works with 2.0.